src/main/api/2/studio-api-2.yaml

```diff
@@ -3860,6 +3860,86 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content/children_by_paths:
+    post:
+      tags:
+        - content
+      summary: Get list of children for given item paths
+      description: 'Required permission "get_children" and "site member"'
+      operationId: getChildrenByPaths
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                siteId:
+                  description: Site ID
+                  type: string
+                localeCode:
+                  description: |-
+                    filter children by locale code, or show source items (not translation) if translation to
+                    specified locale code does not exist
+                  type: string
+                paths:
+                  description: item paths to get children for
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/ChildrenByPathRequest'
+              required:
+                - siteId
+                - paths
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  items:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        path:
+                          type: string
+                        levelDescriptor:
+                          $ref: '#/components/schemas/SandboxItem'
+                        children:
+                          type: array
+                          items:
+                            $ref: '#/components/schemas/SandboxItem'
+                        total:
+                          type: integer
+                          format: int32
+                          description: Total records in the system
+                        offset:
+                          type: integer
+                          format: int32
+                          description: Offset of first record in this response
+                        limit:
+                          type: integer
+                          format: int32
+                          description: Number of records returned in this response
```

Comment:
We send a single call to `sandbox_items_by_path` to fetch all the parents at once, and 1 call per parent to get its children to `children_by_path`. We could save one more request if this new api would return the SandboxItem (parent) for that path too, albeit, I do believe there are cases where we only need to refresh the parent or only need to refresh the children so we call only the `item_by_path` or `children_by_path` alone.